src/xma/src/xmaapi/xmafilter.cpp

```diff
@@ -269,6 +259,38 @@ xma_filter_session_create(XmaFilterProperties *filter_props)
     filter_session->base.hw_session.private_do_not_use = (void*) priv1;
     filter_session->base.session_signature = (void*)(((uint64_t)priv1) | ((uint64_t)priv1->reserved));
 
+    int32_t num_execbo = g_xma_singleton->num_execbos;
```

Comment:
Again duplicated?   Can this be consolidated?

Reply:
change coming in different PR